src/main/java/com/dataloom/datastore/search/controllers/SearchController.java

```diff
@@ -104,4 +104,18 @@ public String executeQueryJson( @RequestBody SearchRequest request ) {
     @Override public AuthorizationManager getAuthorizationManager() {
         return authorizations;
     }
+
+    @RequestMapping(
+            path = { ORGANIZATIONS },
+            method = RequestMethod.POST,
+            produces = { MediaType.APPLICATION_JSON_VALUE } )
+    @Override
+    public String executeOrganizationSearch( @RequestBody String searchTerm ) {
+        try {
+            return ObjectMappers.getJsonMapper().writeValueAsString( searchService.executeOrganizationKeywordSearch( searchTerm ) );
```

Comment:
so that we're returning JSON to the frontend--executeOrganizationKeywordSearch returns a List<Map<String, Object>>